common/api/imodel-components-react.api.md

```diff
@@ -440,6 +439,9 @@ export interface DrawingViewportChangeEventArgs {
     rotation: Matrix3d;
 }
 
+// @public
+export function emitTimelineControlEvent({ uiComponentId, timelineAction, }: TimelinePausePlayArgs): void;
```

Comment:
I understand that the point is to support the previous API, however, it makes little sense to have this sort of declarative API.
Could we simply add a `isPlaying` prop to a component for now and see what issues/complaints we get?

Reply:
I can look into that, sure.

Reply:
That makes sense to me. I left the old deprecated way to avoid breaking and added a note that the two ways are mutually exclusive and should not be used together.

Haven't used class components in a while :)

Reply:
Done